[2024-07-21T02:21:10.170+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-21T02:21:10.190+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: one_task_dag.one_task manual__2024-07-21T02:21:06.296001+00:00 [queued]>
[2024-07-21T02:21:10.194+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: one_task_dag.one_task manual__2024-07-21T02:21:06.296001+00:00 [queued]>
[2024-07-21T02:21:10.195+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-21T02:21:10.225+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): one_task> on 2024-07-21 02:21:06.296001+00:00
[2024-07-21T02:21:10.229+0000] {standard_task_runner.py:63} INFO - Started process 5226 to run task
[2024-07-21T02:21:10.231+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'one_task_dag', 'one_task', 'manual__2024-07-21T02:21:06.296001+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/one_task_dag.py', '--cfg-path', '/tmp/tmpxq2obkum']
[2024-07-21T02:21:10.232+0000] {standard_task_runner.py:91} INFO - Job 8: Subtask one_task
[2024-07-21T02:21:10.283+0000] {task_command.py:426} INFO - Running <TaskInstance: one_task_dag.one_task manual__2024-07-21T02:21:06.296001+00:00 [running]> on host codespaces-8c1ce7
[2024-07-21T02:21:10.370+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Vinoo' AIRFLOW_CTX_DAG_ID='one_task_dag' AIRFLOW_CTX_TASK_ID='one_task' AIRFLOW_CTX_EXECUTION_DATE='2024-07-21T02:21:06.296001+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-21T02:21:06.296001+00:00'
[2024-07-21T02:21:10.371+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-21T02:21:10.372+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-07-21T02:21:10.372+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "hello linkedin learning!" > /workspaces/hands-on-introduction-data-engineering-4395021/lab1/create_this_file.txt']
[2024-07-21T02:21:10.380+0000] {subprocess.py:86} INFO - Output:
[2024-07-21T02:21:10.381+0000] {subprocess.py:93} INFO - /usr/bin/bash: /workspaces/hands-on-introduction-data-engineering-4395021/lab1/create_this_file.txt: No such file or directory
[2024-07-21T02:21:10.381+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2024-07-21T02:21:10.382+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-21T02:21:10.387+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/usr/local/python/3.10.13/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/usr/local/python/3.10.13/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/usr/local/python/3.10.13/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/local/python/3.10.13/lib/python3.10/site-packages/airflow/operators/bash.py", line 243, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-07-21T02:21:10.389+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=one_task_dag, task_id=one_task, run_id=manual__2024-07-21T02:21:06.296001+00:00, execution_date=20240721T022106, start_date=20240721T022110, end_date=20240721T022110
[2024-07-21T02:21:10.417+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 8 for task one_task (Bash command failed. The command returned a non-zero exit code 1.; 5226)
[2024-07-21T02:21:10.444+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-07-21T02:21:10.453+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-21T02:21:10.453+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
